docs/TestCoverage.md

```diff
@@ -7521,7 +7560,45 @@ expect(node, inputs=[a, a_scale, a_zero_point, b, b_scale, b_zero_point, y_scale
 
 
 ### QuantizeLinear
-There are 1 test cases, listed as following:
+There are 2 test cases, listed as following:
+<details>
+<summary>quantizelinear_channels</summary>
+
+```python
+node = onnx.helper.make_node('QuantizeLinear',
+    inputs=['x', 'y_scale', 'y_zero_point'],
+    outputs=['y'],)
+
+x = np.array([[[[-162,   10],
+                [-100,  232],
+                [ -20,  -50]],
+
+               [[ -76,    0],
+                [   0,  252],
+                [  32,  -44]],
+
+               [[ 245, -485],
+                [-960, -270],
+                [-375, -470]],],], dtype=np.float32)
+y_scale = np.array([2, 4, 5], dtype=np.float32)
+y_zero_point = np.array([84, 24, 196], dtype=np.uint8)
+y = np.array([[[[  3,  89],
```

Comment:
Same as above please add a method to compute this output. Adding a method will benefit anybody who is implementing this operator in their runtime.

Reply:
Done!